### PR TITLE
upgrade dependencies, raise version

### PR DESCRIPTION
**Pull request by TomTasche**

### CODE REVIEW

File: app/CMakeUserPresets.json
Comment (TomTasche) [inline app/CMakeUserPresets.json:1]: Shall we gitignore that one?

File: app/build.gradle [hunk 1]
```diff
@@ -26,7 +26,7 @@ task conanInstall {
     ["armv7", "armv8", "x86", "x86_64"].each { String arch ->
         exec {
             commandLine(
-                "conan", "install", ".",
+                "/home/tom/Documents/ODR/OpenDocument.droid/bin/conan", "install", ".",
```
Comment (TomTasche) [inline app/build.gradle:29]: Not sure if we should keep it like this, but at least that makes it obvious what needs to be changed in order to make it work on another system?

Comment (andiwand) [reply app/build.gradle:29]: I fear that will break the CI

File: app/CMakeUserPresets.json
Comment (andiwand) [reply app/CMakeUserPresets.json:1]: makes sense 👍

File: app/build.gradle [hunk 2]
```diff
@@ -25,6 +25,7 @@ task conanProfile {
 task conanInstall {
     ["armv7", "armv8", "x86", "x86_64"].each { String arch ->
         exec {
+            // replace with local path to conan-binary
```
Comment (ViliusSutkus89) [inline app/build.gradle:28]: It's not that I care about this comment, but it makes no sense now :D

Comment (ViliusSutkus89) [inline app/build.gradle:28]: Add conan to your $PATH, and then you won't have to do any path replacing in gradle files

Comment (TomTasche) [reply app/build.gradle]: you're right, thanks!

**Pull request merged**